spec/index.bs

```diff
@@ -4020,34 +4020,34 @@ Issue: link to the semantics of SV_SampleIndex and SV_Coverage in WGSL spec.
 
 <script type=idl>
 dictionary GPUOutputMergerState {
-    required sequence<GPUColorStateDescriptor> colorStates;
+    required sequence<GPUColorOutputState> colors;
```

Comment:
I don't love this name as it's not a list of "colors". Maybe colorAttachments or colorOutputs?

Note depthStencil is not strictly output, and it's singular anyway, so it makes sense to keep that name. Though colorOutputs may not strictly be output either (especially with programmable blending?).

Reply:
sure, let's rename to `colorOutputs`